luigi_tools/parameter.py

```diff
@@ -100,6 +111,8 @@ def __init__(self, *args, absolute=False, create=False, exists=False, **kwargs):
         self.create = create
         self.exists = exists
 
+        luigi_tools.moved_to_luigi_warning(previous_luigi_version="3.0.3")
```

Comment:
Should wait for https://github.com/spotify/luigi/pull/3126 to be merged or closed